tutoriaux/CNN/CNN_a_completer.ipynb

```diff
@@ -1200,21 +1201,20 @@
    },
    "source": [
     "### Batch normalization\n",
-    "Le *batch normalization* est une astuce qui permet, en pratique, au modèle d'apprendre plus vite. Elle agit comme régularisateur en normalisant les entrées par batch, de manière différentiable.\n",
+    "Le *batch normalization* est une astuce qui permet, en pratique, au modèle d'apprendre plus vite. Elle agit comme régularisateur en normalisant les entrées par batch, de manière différentiable. Les données en sorties de cette couche auront une moyenne proche de 0 et une variance proche de 1.\n",
     "\n",
     "![Alt Text](https://github.com/mila-iqia/ecole_dl_mila_ivado_2018_10/blob/master/tutoriaux/CNN/images/1_Hiq-rLFGDpESpr8QNsJ1jg.png?raw=true)\n",
     "\n",
     "\n",
     "Pour plus d'informations sur le *Batch Normalization*, consultez cet [article](https://arxiv.org/pdf/1502.03167v3.pdf).\n",
     "\n",
     "### Boîte à outils\n",
-    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__`. La classe suivante peut être utilisée:\n",
+    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__` et de l'appeler après chaque Relu. La classe suivante peut être utilisée:\n",
     "<ul>\n",
     "<li><a href=\"http://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm2d\">`nn.BatchNorm2d(num_features)`</a> : permet d'ajouter de la batch normalisation à une entrée à 4 dimensions présentée sous la forme d'un tenseur à 3 dimensions.</li>\n",
     "</ul>\n",
     "\n",
-    "### Implémentation\n",
-    "Ci-dessous vous pouvez remarquer que la classe LeNet5 a été modifiée pour y ajouter la batch normalisation."
```

Comment:
Why remove this one ?

Reply:
Because there is nothing to notice, they have to implement it, not look at it.

Reply:
True